breaking/entrypoint.sh

```diff
@@ -16,6 +16,8 @@ echo "running oasdiff breaking... base: $base, revision: $revision, fail_on_diff
 flags=""
 if [ "$fail_on_diff" = "true" ]; then
     flags="${flags} --fail-on WARN"
+elif [ "$fail_on_diff" != "false" ]
+    flags="${flags} --fail-on $fail_on_diff"
```

Comment:
I don't think this is a best practice.

Reply:
@effoeffi Is there a way to indicate in the `oasdiff breaking change action` to only `fail` on `ERR` instead of `WARN`?

Reply:
I understand, but it is not best practice to use 'var' as both a boolean and a string. I suggest converting it to a string similar to `oasdiff`. Additionally, update the documentation, defaults, and create tests accordingly. Please also review other parameters for similar improvements.

Reply:
@effoeffi Updated the `action`, `README`, etc. in  `oasdiff-action` in d96dd2c64ce60803260888512b31a40363c37c68. Which style tests are in the repo or would this be the first tests in the repo if added?

Reply:
@effoeffi Added a test in https://github.com/oasdiff/oasdiff-action/pull/43/commits/468ff610dbcaff306009a416dc7bf5a849ad40e1.